Sources/App/Core/SearchFilter/Filters/KeywordSearchFilter.swift

```diff
@@ -33,8 +35,13 @@ struct KeywordSearchFilter: SearchFilterProtocol {
         self.predicate = .init(
             operator: (expression.operator == .is) ?
                 .caseInsensitiveLike : .notCaseInsensitiveLike,
-            bindableValue: .value("%\(expression.value)%"),
+            bindableValue: .value(expression.value),
             displayValue: expression.value
         )
     }
+
+    var sqlIdentifier: SQLExpression {
+        // override default identifier
+        arrayToString(Search.keywords, delimiter: " ")
```

Comment:
This actually only works for single keywords, which is why our test passes. I.e.

```sql
select ARRAY_TO_STRING(ARRAY['foo'], ' ') ILIKE 'foo';
```
→ TRUE

```
select ARRAY_TO_STRING(ARRAY['foo', 'bar'], ' ') ILIKE 'foo';
```
→ FALSE

which makes sense.

Reply:
I've updated `test_keywordSearchFilter` to expose this in 76101d78c72faf2696cf073717dca44202eb5798